src/Toml/Bi/Map.hs

```diff
@@ -572,6 +578,26 @@ _IntSet :: TomlBiMap IS.IntSet AnyValue
 _IntSet = iso IS.toList IS.fromList >>> _Array _Int
 {-# INLINE _IntSet #-}
 
+_KeyText :: TomlBiMap Key Text
+_KeyText = BiMap
+    { forward = Right . keyToText
+    , backward = textToKey
+    }
+
+_KeyString :: TomlBiMap Key String
+_KeyString = BiMap
+    { forward = Right . T.unpack . keyToText
+    , backward = textToKey . T.pack
+    }
+
+keyToText :: Key -> Text
+keyToText = (T.intercalate ".") . (map unPiece) . toList . unKey
```

Comment:
This looks like a useful function in general. Could you, please move into the `PrefixTree` module after this line?

https://github.com/kowainik/tomland/blob/6ca6ed3a3d6e1bbcd2b7b06ac82238ef56b00ccc/src/Toml/PrefixTree.hs#L105

Also, I think it can be written in a slightly shorter and more performant way, using `coerce` trick:

```suggestion
keyToText :: Key -> Text
keyToText = T.intercalate "." . toList . coerce
```

`coerce` here will automatically convert `Key` to `NonEmpty Text` with zero runtime cost.